### PR TITLE
add copy to clipboard and source link

### LINKED ISSUE #159: installers should show "view source" option

### PR DESCRIPTION
**Pull request by SaraVieira**
- Adds a copy to clipboard button on the homepage to the curl scripts
- Downloads the installer files, adds `txt` extension to them
- Adds links to same file

image

https://user-images.githubusercontent.com/1051509/217678400-c104f6bc-d55d-4ed1-bbe1-de067e20ef8e.mp4

Would perfer to make a new PR to add the same thing to the downloads page since this PR is already big but open for ideas there

### CODE REVIEW

File: src/site/layout/mod.rs [hunk 1]
```diff
@@ -61,8 +61,9 @@ pub fn build(config: &Config, content: String, is_index: bool) -> Result<String>
             {homepage}
             {favicon}
             {meta_tags}
-            {fringe_css}
+            // {fringe_css}
             {additional_css}
+            <link href="http://localhost:42673/axo-oranda.css" rel="stylesheet" />
```
Comment (ashleygwilliams) [inline src/site/layout/mod.rs]: lol

File: oranda.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "favicon": "https://www.axo.dev/favicon.ico",
-  "path_prefix": "oranda",
```
Comment (SaraVieira) [inline oranda.json]: FIX BEFORE MERGE

File: src/site/layout/mod.rs (hunk 1 above)
Comment (SaraVieira) [inline src/site/layout/mod.rs]: FIX BEFORE MERGE

**Review by ashleygwilliams — APPROVED** [review]
looks pretty good, gotta fix this dev exp tho heh

**Pull request merged**